src/components/network/network-table.js

```diff
@@ -1004,7 +1004,12 @@ const NetworkTable = (props) => {
         (rowData, columnDefinition, key, style, rowIndex) => {
             if (
                 isLineOnEditMode(rowData) &&
-                rowData[columnDefinition.dataKey] !== undefined &&
+                (rowData[columnDefinition.dataKey] !== undefined ||
```

Comment:
it will be more readable if we externalize those conditions into a function

Reply:
Done